IRCT-CL/src/main/java/edu/harvard/hms/dbmi/bd2k/irct/cl/filter/CORSFilter.java

```diff
@@ -0,0 +1,23 @@
+package edu.harvard.hms.dbmi.bd2k.irct.cl.filter;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class CORSFilter implements ContainerResponseFilter {
+
+    @javax.annotation.Resource(mappedName = "java:global/cors_allow_origin")
+    private String corsAllowOrigin;
```

Comment:
This annotation will stop the server during start up when the name binding is missing, which means the default situation will need user to put value in this paramter, but in default, we'd like to disable Cross-Origin Resource Sharing. To implement not interrupting startup procedure, you can take ideas from this code: [https://github.com/hms-dbmi/IRCT/blob/master/IRCT-API/src/main/java/edu/harvard/hms/dbmi/bd2k/irct/IRCTApplication.java#L469](https://github.com/hms-dbmi/IRCT/blob/master/IRCT-API/src/main/java/edu/harvard/hms/dbmi/bd2k/irct/IRCTApplication.java#L469)

Reply:
Thanks for the feedback, see the most recent commit which should address the issue.
Anything else problematic?